packages/react-import-remote/src/test/ImportRemote.test.tsx

```diff
@@ -102,9 +105,13 @@ describe('<ImportRemote />', () => {
       const spy = jest.fn();
       load.mockImplementation(() => promise);
 
-      mount(<ImportRemote {...mockProps} onImported={spy} />);
+      const importRemote = await mount(
+        <ImportRemote {...mockProps} onImported={spy} />,
+      );
 
-      await promise;
+      importRemote.act(async () => {
```

Comment:
Hmm async `act` should not work yet...

Reply:
whatever this is doing it's not doing what you think so we should probably just skip it until async act is actually real.

Reply:
Please delete it or make it not be async.